TraderJoe-Bots/Re-entrancy-Transfers/src/agent.spec.ts

```diff
@@ -0,0 +1,400 @@
+import { BigNumber, Transaction } from "ethers";
+import { Interface } from "ethers/lib/utils";
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  createTransactionEvent,
+  TransactionEvent,
+  Trace,
+  TraceAction,
+} from "forta-agent";
+import { MockEthersProvider } from "forta-agent-tools/lib/mock.utils";
+import { createAddress, TestTransactionEvent } from "forta-agent-tools/lib/tests.utils";
+import { provideHandleTransaction } from "./agent";
+import MarketsFetcher from "./markets.fetcher";
+import NetworkData from "./network";
+import { FUNCTIONS_IFACE, FUNCTIONS_MAP, MARKET_UPDATE_ABIS } from "./utils";
+
+describe("Re-entrancy transfers bot test suite", () => {
+  let handleTransaction: HandleTransaction;
+
+  const USER_ADDR = createAddress("0xc1");
+  const TEST_MARKETS = [createAddress("0x11"), createAddress("0x22"), createAddress("0x33")];
+
+  const mockNetworkManager: NetworkData = {
+    joeTroller: createAddress("0x0ae"),
+    sJoeStaking: createAddress("0x1ae"),
+    masterChefV2: createAddress("0x2ae"),
+    moneyMaker: createAddress("0x3ae"),
+    networkMap: {},
+    setNetwork: jest.fn(),
+  };
+  const mockProvider = new MockEthersProvider();
+  const mockFetcher = new MarketsFetcher(mockProvider as any);
+  const mockUpdateMarkets = jest.fn();
+
+  const createTrace = (from: string, traceAddress: number[], to: string, input: string): Trace => {
+    return {
+      action: {
+        from,
+        to,
+        input,
+      } as TraceAction,
+      traceAddress,
+    } as Trace;
+  };
+
+  const createFinding = (
+    from: string,
+    address: string,
+    rentrancyFrom: string,
+    initialCall: string,
+    reEtrantCall: string
```

Comment:
```suggestion
    reentrancyFrom: string,
    initialCall: string,
    reEntrantCall: string
```
I would even use `reentrant` as a word, I mean in the last param I would not use that capital E